lib/src/kvs/tests/clock.rs

```diff
@@ -0,0 +1,8 @@
+use crate::kvs::clock::SystemClock;
+
+#[tokio::test]
+#[serial]
+async fn get_clock_now() {
```

Comment:
This can be part of `lib/src/kvs/clock.rs` under
```
#[cfg(tests)]
mod test {
...
}
```

Reply:
Added at the bottom